public/app/core/components/FormPrompt/FormPrompt.tsx

```diff
@@ -0,0 +1,117 @@
+import { css } from '@emotion/css';
```

Comment:
This is a copy of the component from Enterprise. It'd be removed from there and this version should be used instead.